barcode/writer.py

```diff
@@ -322,11 +327,14 @@ def _paint_module(self, xpos, ypos, width, color):
             self._draw.rectangle(size, outline=color, fill=color)
 
         def _paint_text(self, xpos, ypos):
-            font = ImageFont.truetype(FONT, self.font_size * 2)
-            width, height = font.getsize(self.text)
-            pos = (mm2px(xpos, self.dpi) - width // 2,
-                   mm2px(ypos, self.dpi) - height // 4)
-            self._draw.text(pos, self.text, font=font, fill=self.foreground)
+            for subtext in self.text.split('\n'):
```

Comment:
I'll assume you visually inspected the output of these?
(we don't have a very good tests suite yet :disappointed:  )

Reply:
For png the output looks good.  For svg the output has a too small vertical size (if you have more than 3 lines) and I could not find a quick way to fix it. Maybe you have some ideas there.

Reply:
Hmmm. I guess it's better than the current situation, and that can be tweaked later as well.